content/posts/2019-05-31-continuous-deployment-for-hugo-with-aws-amplify/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Continuous Deployment for Hugo With AWS Amplify
+description: 'Learn how to build a Continuous Delivery pipeline for Hugo, using the AWS Amplify Console'
+banner: './banner.png'
+authorIds:
+  - chris-mcquaid
+href: https://thetestlabs.io/post/continuous-deployment-for-hugo-with-aws-amplify/
+platforms:
+  - Hugo
+categories:
+  - Hosting
+  - Continuous Deployment
```

Comment:
Can you please remove the continue deployment tag? We are keeping the categories in sync with the Amplify ones